test/cmd.js

```diff
@@ -1,4 +1,5 @@
 
+require("babel-core/register");
```

Comment:
We need to do something different here; this will cause the tests not to detect an accidental ES6 syntax in the ES5 apps.